README.md

```diff
@@ -73,6 +73,36 @@ in rust that supports proper month/years holding and literal representation. You
 (5) we are going to implement types that both provide stored value as integer and
 support proper (de)serialization
 
+## `any` elements handling
+
+There are cases when schema allows extensions for the certain type.
+
+```xml
+<xs:complexType name="MyType">
+    <xs:sequence>
+        <xs:element name="Parameters" type="xs:string" />
+        <xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>
+    </xs:sequence>
+    <xs:anyAttribute namespace="##any" processContents="lax"/>
+</xs:complexType>
+```
+
+In such cases we don't know in advance what fields must be present in Rust struct so we don't add them to output:
+
+```rust
+#[derive(Default, PartialEq, Debug, YaSerialize, YaDeserialize)]
+#[yaserde(prefix = "tns", namespace = "tns: http://example.com")]
+pub struct MyType {
+    #[yaserde(prefix = "tns", rename = "Parameters")]
+    pub parameters: String,
+}
+```
+
+In this unlucky situation to support extensions user can either:
+- modify the generated code and add fields wrapped into `Option` manually
+- patch source XSD to add needed elements there (with `minOccurs="0"`)
+- [unimplemented] pass a dictionary to xsd-parser-rs to insert extension fields into certain types
```

Comment:
Could you add this as a point in a list of implemented/unimplemented features too?